LowDiscrepancySequenceImplementation_doc.i.in/examples/data_analysis/README.txt

```diff
@@ -0,0 +1,4 @@
+.. _Data analysis:
+
```

Comment:
there seems to be a problem with this erroneous directory '[LowDiscrepancySequenceImplementation_doc.i.in/'  which contains the html site

Reply:
I'm sorry: I do not undertsand the problem. I checked with the file of the master: it seems the same! Furthermore, I did not modify the file python/doc/examples/data_analysis/README.txt

Reply:
You added a new directory named "LowDiscrepancySequenceImplementation_doc.i.in" in the root dir, remove it

Reply:
@jschueller Thank you!!!! Could you precise me the command? Is it 
'git remove LowDiscrepancySequenceImplementation_doc.i.in'
?

Reply:
nope, I dont see anything pushed, the files are still there

Reply:
ok... which commands ? I really don't know...

Reply:
if its marked D you deleted but not commited yet

Reply:
ok... I added the deleted files.... it is ok now! :-)